Makefile

```diff
@@ -78,6 +78,10 @@ start: ## Start the environment
 		--sync-mode="two-way-resolved" \
 		--ignore-vcs --ignore=".idea" \
 		"${PROJECT_LOCATION}" "docker://${COMPOSE_PROJECT_NAME}_synchro/var/www/html/"
+	while [[ ! $$(mutagen list) =~ 'Status: Watching for changes' ]]; do \
+		echo 'Waiting for synchronization to complete...'; \
+		sleep 10; \
```

Comment:
Isn't 10 seconds a bit long ?

Reply:
The initial synchronization takes between 2 and 3 minutes to complete. It will increase the spam if I reduce the sleep duration. 😄

Reply:
And the restart process takes between 40 and 50 seconds.

Reply:
Okay, good enough for me :)